### PR TITLE
Add new query parameter to snap search

### PR DESCRIPTION
**Pull request by steverydz**
## Done
Added a new query parameter to the snaps search

### CODE REVIEW

File: canonicalwebteam/store_api/stores/snapstore.py [hunk 1]
```diff
@@ -125,12 +125,17 @@ def get_store(self, session, store_id):
...
-    def get_store_snaps(self, session, store_id, query=None):
+    def get_store_snaps(
+        self, session, store_id, query=None, allowed_for_inclusion=None
+    ):
         headers = self._get_authorization_header(session)
         params = {}
 
         if query:
-            params = {"q": query}
+            params["q"] = query
```
Comment (jkfran) [inline canonicalwebteam/store_api/stores/snapstore.py:135]: Nice!

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = 'canonicalwebteam.store-api'
-version = '3.1.8'
+version = '3.2.0'
```
Comment (jkfran) [inline pyproject.toml:3]: You don't like 3.1.9?  :P

File: canonicalwebteam/store_api/stores/snapstore.py (hunk 1 above)
Comment (steverydz) [reply canonicalwebteam/store_api/stores/snapstore.py:135]: sorry, fixed

File: pyproject.toml (hunk 2 above)
Comment (steverydz) [reply pyproject.toml]: sorry, fixed

**Review by jkfran — APPROVED** [review]
Looks good!

**Pull request merged**